tls/s2n_x509_validator.c

```diff
@@ -165,6 +166,7 @@ int s2n_x509_validator_init(struct s2n_x509_validator *validator, struct s2n_x50
         POSIX_ENSURE_REF(validator->store_ctx);
     }
     validator->cert_chain_from_wire = sk_X509_new_null();
+    validator->cert_chain_validated = NULL;
     validator->state = INIT;
```

Comment:
If wipe can handle NULLs, do the wipe here too just to make sure you don't leak (and everywhere else you set this to NULL).